### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by ngdangtu-vn**
It's good idea to tell people that the project also support Deno.

### CODE REVIEW

**Comment by ngdangtu-vn** [comment]
It works well so far. I don't see any incompatibility between the app and Deno.

File: README.md [hunk 1]
```diff
@@ -21,10 +21,19 @@ To get things done **Minify** uses this amazing tools:
...
+
+```sh
 npm i minify -g
 ```
 
+For Deno users
+
+```js
+// inside Js/Ts file
+import { minify } from "npm:minify";
```
Comment (coderaiser) [inline README.md:34]:
```suggestion
import {minify} from "npm:minify";
```

File: README.md [hunk 2]
```diff
@@ -21,10 +21,19 @@ To get things done **Minify** uses this amazing tools:
...
+For Node users
+
+```sh
 npm i minify -g
 ```
 
+For Deno users
+
+```js
+// inside Js/Ts file
```
Comment (coderaiser) [inline README.md:33]:
```suggestion
```

**Comment by coderaiser** [comment]
Thanks!

**Pull request merged**